package.json

```diff
@@ -46,7 +46,10 @@
   },
   "homepage": "https://github.com/ericclemmons/npm-install-webpack-plugin#readme",
   "dependencies": {
+    "@cycle/core": "^7.0.0",
+    "babel-loader": "^8.2.2",
     "cross-spawn": "^5.0.1",
+    "express": "^4.17.1",
```

Comment:
I think we don't need to add deps here when we test, change cwd to tests :smile: and put `package.json` in tests

Reply:
oh this is my bad, fixing 👍

Reply:
Fixed @alexander-akait